src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.g.cs

```diff
@@ -48,7 +48,7 @@ private static ImmutableArray<SuppressionDescriptor> GetDescriptors()
                 // If the number of diagnostic ids changes significantly or decreases when the analysers are updated,
                 // investigate! There may be a problem with code generator.
                 // *************************************************************************************************
-                // Number of unique diagnostic ids (C# and VB.NET): 435
+                // Number of unique diagnostic ids (C# and VB.NET): 448
```

Comment:
We currently have to remember to manually trigger regenerating this file each time the analyzer is updated, but we don't always (it is documented on the [extranet](https://xtranet-sonarsource.atlassian.net/wiki/spaces/SL/pages/2667413623/Update+Embedded+SonarAnalyzers+SLVS) and there is a [ticket](https://github.com/SonarSource/sonarlint-visualstudio/issues/3335) for it).